be/src/vec/functions/function_jsonb.cpp

```diff
@@ -526,8 +523,13 @@ struct JsonbExtractStringImpl {
             int r_size = roffsets[i] - roffsets[i - 1];
             const char* r_raw = reinterpret_cast<const char*>(&rdata[roffsets[i - 1]]);
 
+            JsonbPath path;
+            if (!path.seek(r_raw, r_size)) {
+                is_invalid_json_path = true;
```

Comment:
if is_invalid_json_path = true , whether we do not need to call inner_loop_impl func ,and just push_null_string & return ?